sysutils/defaultbrowser/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+PortGroup       makefile 1.0
+
+github.setup    kerma defaultbrowser 1.1
+revision        0
+categories      sysutils
+maintainers     {@i0ntempest me.com:szf1234} openmaintainer
+platforms       darwin
+license         MIT
+
+description     Command line tool for setting the default browser (HTTP handler) in macOS
+long_description \
+                ${description}
+
+homepage        https://github.com/kerma/defaultbrowser
+
+pre-build {
+    reinplace "s|install: all|install:|" ${worksrcpath}/Makefile
+    reinplace "s|gcc -o|\$(CC) -o|" ${worksrcpath}/Makefile
+}
+
+destroot.args   PREFIX=${destroot}${prefix}
```

Comment:
The makefile portgroup should have a way of handling this for you too, if not automatically then through as setting.